scripts/strip.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+#
+# Script to remove references to .symtab from relocation section and then run strip
+#
+# Copyright 2022 Phoenix Systems
+# Author: Andrzej Glowinski
+#
+
+import shutil
+import subprocess
+import sys
+import tempfile
+from io import BytesIO
+from dataclasses import dataclass
+from enum import IntEnum
+import struct
+from typing import ClassVar, Type
+
+
+class EiClass(IntEnum):
+    """EI_CLASS field from e_ident that determines if elf is 32 or 64-bit"""
+    ELFCLASSNONE = 0
+    ELFCLASS32 = 1
+    ELFCLASS64 = 2
+
+
+class EiData(IntEnum):
+    """EI_DATA field from e_ident that determines if elf is little or big endian"""
+    ELFDATANONE = 0
+    ELFDATA2LSB = 1
+    ELFDATA2MSB = 2
+
+    def to_format(self):
+        return {EiData.ELFDATA2LSB: "<", EiData.ELFDATA2MSB: ">"}[self]
+
+
+# Types only needed for resolving relocations
+class ShType(IntEnum):
+    """Interpretation of sh_type field of ElfXX_Shdr struct. Determines section type"""
+    SHT_NULL = 0
+    SHT_RELA = 4
+    SHT_REL = 9
+
+
+class ElfStruct:
+    """Abstract for every ELF struct"""
+    FORMAT: ClassVar[str]
```

Comment:
```suggestion
    FORMAT: ClassVar[List[Tuple[str, str]]]
```

Reply:
Thanks, my typechecker not complaining on it